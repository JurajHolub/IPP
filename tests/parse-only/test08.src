.IPPcode19
DEFVAR GF@var
PUSHS int@65
POPS GF@var
DEFVAR GF@res
INT2CHAR GF@res GF@var
WRITE GF@res
INT2CHAR GF@res int@66
WRITE GF@res
