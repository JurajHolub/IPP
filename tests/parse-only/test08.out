<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
  <instruction order="1" opcode="DEFVAR">
    <arg1 type="var">GF@var</arg1>
  </instruction>
  <instruction order="2" opcode="PUSHS">
    <arg1 type="int">65</arg1>
  </instruction>
  <instruction order="3" opcode="POPS">
    <arg1 type="var">GF@var</arg1>
  </instruction>
  <instruction order="4" opcode="DEFVAR">
    <arg1 type="var">GF@res</arg1>
  </instruction>
  <instruction order="5" opcode="INT2CHAR">
    <arg1 type="var">GF@res</arg1>
    <arg2 type="var">GF@var</arg2>
  </instruction>
  <instruction order="6" opcode="WRITE">
    <arg1 type="var">GF@res</arg1>
  </instruction>
  <instruction order="7" opcode="INT2CHAR">
    <arg1 type="var">GF@res</arg1>
    <arg2 type="int">66</arg2>
  </instruction>
  <instruction order="8" opcode="WRITE">
    <arg1 type="var">GF@res</arg1>
  </instruction>
</program>
